Remove xlarge-* references from admin forms

alecslupu opened the pull request:
<!--
NOTE: We are in the middle of a big redesign of the frontend.
That could mean that your PR will not get through on the next few months.
Please see https://github.com/decidim/decidim/discussions/9512
-->

#### :tophat: What? Why?
This PR attempts to clean up the HTML markup in admin by removing useless class xlarge-*

#### Testing
1. Login as admin go to admin panel 
2. From the files section try to find the changed forms 
3. See there are no changes regarding UI

:hearts: Thank you!

andreslucena reviewed (commented):
I was just skiming this PR and I don't like the approach as we're mixing multiple problems in the same PR: 

* the dead CSS classes (explained at #11752) 
* the problem with legacy HTML markup structure
* some indentation changes (related with #11707) 

I'd prefer to handle each of these issues in different PRs, as if we're mixing problems/solutions, then these PRs become huge and messy. These already are medium/big but if we keep adding things then the review is just too much work. 

I believe that if we have PRs like #11707 (that goes to a linter configuration change) then that should be easy-peasy to review/merge and to not bring noise to the rest of PRs. 

What do you think @alecslupu?

alecslupu commented:
> I was just skiming this PR and I don't like the approach as we're mixing multiple problems in the same PR:
> 
>     * the dead CSS classes (explained at [Old CSS classes still in the views #11752](https://github.com/decidim/decidim/issues/11752))
> 
>     * the problem with legacy HTML markup structure
> 
>     * some indentation changes (related with [Apply new rubocop rules on erb - Argument identation #11707](https://github.com/decidim/decidim/pull/11707))
> 
> 
> I'd prefer to handle each of these issues in different PRs, as if we're mixing problems/solutions, then these PRs become huge and messy. These already are medium/big but if we keep adding things then the review is just too much work.
> 
> I believe that if we have PRs like #11707 (that goes to a linter configuration change) then that should be easy-peasy to review/merge and to not bring noise to the rest of PRs.
> 
> What do you think @alecslupu?

@andreslucena you are right. I have refactored this PR. Since this is draft, i assume you have not really started to review it :D

i have added the ERB syntax as well for deprecated classes.

andreslucena reviewed (approved):
![cheff kiss gif](https://github.com/decidim/decidim/assets/717367/0724e826-e230-4897-9192-bfee0771b814)

alecslupu commented:
Merging as per @andreslucena 's approval.
